### DISCUSSION TITLE
Streaming completions?

### DISCUSSION THREAD

**Discussion by ctclements** [betalgo/openai#47] [General]
Does streaming work for Completions? I get a JSON reader error every time I set stream to true in my CompletionCreateRequest.

```
var completionResult = await openAIService.Completions.CreateCompletion(new CompletionCreateRequest()
{
	Prompt = "a prompt",
	Temperature = 0,
	MaxTokens = 256,
	BestOf = 1,
	Stream = true
}, GPT3Model);
```

**Comment by kayhantolga** [comment]
I couldn't manage the test this feature while I was working on it. We need to find out what the cause of the problem is.
